source/agora/consensus/EnrollmentManager.d

```diff
@@ -687,13 +687,31 @@ public class EnrollmentManager
             last_height = the latest block height
             block = the block to update the validator set with
             finder = the delegate to find UTXOs with
+            node_utxos = the UTXOs belonging to this node
 
     ***************************************************************************/
 
     public void restoreValidators (ulong last_height, const ref Block block,
-        scope UTXOFinder finder) @safe nothrow
+        scope UTXOFinder finder, const ref UTXOSetValue[Hash] node_utxos)
```

Comment:
```suggestion
        scope UTXOFinder finder, const ref UTXOSetValue[Hash] self_utxos)
```

That should make it slightly clearer IMO (but let me know if you disagree).

Reply:
> scope UTXOFinder finder, const ref UTXOSetValue[Hash] self_utxos)

I guess it's clearer.